### PR TITLE
v76.1.0

### PR DESCRIPTION
**Pull request by ianegordon**
Release PR

DO NOT SUBMIT

### CODE REVIEW

**Comment by ianegordon** [comment]
Verified all committers / authors have signed the CLA.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,49 @@
+# 76.1.0
+
+Add additional snapshot tests and mark outdated theming API as deprecated.
```
Comment (romoore) [inline CHANGELOG.md:3]: Tip: I don't know that we should call these APIs "outdated", we are just moving toward a newer approach.  Maybe the more neutral, "...and marking Dialogs Themer APIs as deprecated." would work?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,49 @@
...
+
+## New deprecations
+
+The previous theming API is being deprecated in favor of MDCContainerScheme.
```
Comment (romoore) [inline CHANGELOG.md:7]: In favor of Theming Extensions? Maybe link to the docs on GitHub?

**Review by romoore — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (romoore) [reply CHANGELOG.md:7]: Docs here: 
https://github.com/material-components/material-components-ios/blob/develop/components/schemes/README.md

Comment (romoore) [reply CHANGELOG.md:7]: Why wasn't this feedback addressed before merging?

File: CHANGELOG.md (hunk 1 above)
Comment (ianegordon) [reply CHANGELOG.md:3]: I think the newer approach validates calling the old API outdated, but I can definitely see the negative connotation.
s/outdated/previous/

File: CHANGELOG.md (hunk 2 above)
Comment (ianegordon) [reply CHANGELOG.md:7]: Follow-up PR incoming.

**Pull request merged**